ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -278,9 +296,9 @@ func createCluster(context *cli.Context, awsClients *AWSClients, commandConfig *
 		}
 	}
 	// Create cfn stack
-	template := cloudformation.GetTemplate()
+	template := cloudformation.GetClusterTemplate()
 
-	if _, err := cfnClient.CreateStack(template, stackName, cfnParams); err != nil {
+	if _, err := cfnClient.CreateStack(template, stackName, true, cfnParams); err != nil {
```

Comment:
Could you clarify why we're adding the CapabilityIAM as an argument here?

Reply:
CapabilityIAM is needed when you are creating IAM resources. For cluster creation, we do that, for service discovery we do not. So for SD I want it to be false, so that the CFN template will not run with more permissions then is needed.

Reply:
Ah, got it. Do you think you could add that description to the commit message, for Posterity™️ ? I think you can squash that commit with the one where you update the tests as well.